src/GraphQL/DI/GraphQLBuilderBase.cs

```diff
@@ -18,19 +18,28 @@ public abstract class GraphQLBuilderBase : IGraphQLBuilder
         /// Includes graph types required for connection builders (GraphQL Relay) and generic graph types
         /// such as <see cref="EnumerationGraphType{TEnum}"/> and <see cref="AutoRegisteringObjectGraphType{TSourceType}"/>.
         /// <br/><br/>
-        /// Does not include <see cref="IDocumentWriter"/>, and the default <see cref="IDocumentExecuter"/>
+        /// Does not include <see cref="IGraphQLSerializer"/>, and the default <see cref="IDocumentExecuter"/>
         /// implementation does not support subscriptions.
         /// </summary>
         protected virtual void RegisterDefaultServices()
         {
-            // configure an error to be displayed when no IDocumentWriter is registered
-            Services.TryRegister<IDocumentWriter>(_ =>
+            // configure an error to be displayed when no IGraphQLSerializer is registered
+            Services.TryRegister<IGraphQLSerializer>(_ =>
             {
                 throw new InvalidOperationException(
-                    "IDocumentWriter not set in DI container. " +
-                    "Add a IDocumentWriter implementation, for example " +
-                    "GraphQL.SystemTextJson.DocumentWriter or GraphQL.NewtonsoftJson.DocumentWriter. " +
-                    "For more information, see: https://github.com/graphql-dotnet/graphql-dotnet/blob/master/README.md and https://github.com/graphql-dotnet/server/blob/develop/README.md.");
+                    "IGraphQLSerializer not set in DI container. " +
+                    "Add a IGraphQLSerializer implementation, for example " +
+                    "GraphQL.SystemTextJson.GraphQLSerializer or GraphQL.NewtonsoftJson.GraphQLSerializer. " +
+                    "For more information, see: https://github.com/graphql-dotnet/graphql-dotnet/blob/master/README.md.");
```

Comment:
I would not say that there is enough information in readme now so that the beginner understand what to do next.

Reply:
I agree.  I can retain the links to the server project if you think that's better.  But at some point I think the code in this project should point to docs for itself, and we can link documentation to the server project where it makes sense.  Lmk.

Reply:
>  I can retain the links to the server project if you think that's better. 

No. I'm fine with current error message. It was just a note.